### PR TITLE
Fix Documentation build

### PR DESCRIPTION
**Pull request by chaithyagr**
I have a local passing build with this.
@AGrigis can we please enable build for all p pull requests to ensure we have passing documentation in future?

Adding @zaccharieramzi for review. 
Here is the passing doc page : https://pysap-mri-chaithya.readthedocs.io/en/fixdoc/

### CODE REVIEW

File: mri/gridsearch.py [hunk 1]
```diff
@@ -16,7 +16,6 @@
 import psutil
 
 # Package import
-from mri.numerics.utils import generate_operators
```
Comment (zaccharieramzi) [inline mri/gridsearch.py]: Is this change needed for the documentation to be fixed?
If so why?

File: mri/reconstructors/single_channel.py [hunk 2]
```diff
@@ -17,16 +17,19 @@
...
 
-    For the Synthesis case, finds the solution of:
-        (1/2) * sum(||F Wt alpha - y||^2_2, 1) + mu * H (alpha)
+    Notes
+    -----
+        For the Analysis case, finds the solution  for x of:
+        ..math:: (1/2) * sum(||F x - y||^2_2, 1) + mu * H (W x)
+
+        For the Synthesis case, finds the solution of:
+        ..math:: (1/2) * sum(||F Wt alpha - y||^2_2, 1) + mu * H (alpha)
```
Comment (zaccharieramzi) [inline mri/reconstructors/single_channel.py]: are all these changes needed for the documentation to be built?

File: mri/gridsearch.py (hunk 1 above)
Comment (chaithyagr) [reply mri/gridsearch.py]: Yes, we no longer have this import, and in trying to figure out all modules, documentation errors out.

File: mri/reconstructors/single_channel.py (hunk 2 above)
Comment (chaithyagr) [reply mri/reconstructors/single_channel.py]: .math changes, no, but otherwise these changes weren't showing up in documentation page. Finally, all the white line changes are needed for allowing documentation to build successfully, it's basically white line before parameters etc.

File: mri/gridsearch.py (hunk 1 above)
Comment (zaccharieramzi) [reply mri/gridsearch.py]: I see

File: mri/reconstructors/single_channel.py (hunk 2 above)
Comment (zaccharieramzi) [reply mri/reconstructors/single_channel.py]: fair
